dwave/system/samplers/clique.py

```diff
@@ -96,11 +94,6 @@ def sample(self, bqm, **parameters):
             max_coupling_range = max(total_coupling_range.values())
 
             if (min_coupling_range < min_lim or max_coupling_range > max_lim):
-                warnings.warn(
```

Comment:
Is raising this warning a problem in py312, or is testing it a problem?

It certainly seems useful to keep it, as it might be the only indication to the user that their problem was scaled. (I.e. if they provide a BQM in Binary space, they won't notice an Ising copy was scaled.)

Alternatively, document the scaling happens in the `DWaveCliqueSampler` docs. Nobody will see docs of this private class.

Reply:
The problem that is in 3.12 there are other warnings raised so the tests fail because they check that there is exactly 1.

Re the scaling - the class always scales by default so the fact that it scales according to the per qubit coupling range in addition to the j range is IMO much more of a technical detail than something we need to warn about.